src/main/java/com/github/igotyou/FactoryMod/interactionManager/FurnCraftChestInteractionManager.java

```diff
@@ -217,7 +217,7 @@ public void clicked(Player arg0) {
 		    "Select a recipe");
 	    for (IRecipe rec : fccf.getRecipes()) {
 		InputRecipe recipe = (InputRecipe) (rec);
-		ItemStack recStack = recipe.getRecipeRepresentation();
+		ItemStack recStack = recipe.getRecipeRepresentation().clone();
```

Comment:
If the underlying getRecipeRepresentation method is already returning a clone, why clone again?

Reply:
I did this first, then went through all calls of ISUtils.addLore and ended up also cloning in the getter at the very end, to make everything super safe. That makes this redundant though, you're right, it's not needed.

Reply:
Fixed and squashed